azure-pipelines.yml

```diff
@@ -26,8 +26,8 @@ jobs:
 
   - bash: |
       source activate myEnvironment
-      python setup.py install
-      pytest
+      pip install -e .[dev]
+      pytest --random-order
```

Comment:
Is there a way to specify this flag globally somewhere? If we want to change it later (say if we add other options), it might be easier to do that in one place.

Reply:
@vineetbansal made the same suggestion but gave it up. He did some quick test using the environment variable but can lead to crash. Might be due to the one defined pytest.ini for end users?